+++ src/Makefile.am	2017-12-23 00:17:37.537364707 +0200
@@ -2216,6 +2216,19 @@
 module_allow_passthrough_la_LIBADD = $(MODULE_LIBADD)
 module_allow_passthrough_la_CFLAGS = $(AM_CFLAGS)
 
+
+###################################
+#     Android OpenSL ES sink      #
+###################################
+
+modlibexec_LTLIBRARIES += \
+		module-sles-sink.la
+SYMDEF_FILES += \
+		module-sles-sink-symdef.h
+module_sles_sink_la_SOURCES = modules/module-sles-sink.c
+module_sles_sink_la_LDFLAGS = $(MODULE_LDFLAGS) -lOpenSLES
+module_sles_sink_la_LIBADD = $(MODULE_LIBADD)
+
 ###################################
 #        Some minor stuff         #
 ###################################
